src/modinfo.c

```diff
@@ -1242,16 +1242,69 @@ sr_oper_data_merge_cb(struct lyd_node *trg_node, const struct lyd_node *src_node
     return LY_SUCCESS;
 }
 
+/**
+ * @brief Check whether the session has pushed any operational data to this module.
+ *        If data exists, load it from cache if available.
+ *
+ * @param[in] sess Session to check.
+ * @param[in] mod_name module name to search.
+ * @param[in] dup Flag to duplicate data and not consume it.
+ * @param[out] has_data Flag whether session has push operational data for this module.
+ * @param[out] mod_data Cached module data if available.
+ * @return err_info if an error occurs during duplication, NULL otherwise.
+ */
+static sr_error_info_t *
+sr_modinfo_module_data_cache_load(sr_session_ctx_t *sess, const char *mod_name, int dup, int *has_data, struct lyd_node **mod_data)
```

Comment:
Just a minor improvement, I would prefer some other word than `load`, which is generally used when actually parsing data using DS plugins. Maybe just `get` would be clearer.